examples/passing/RecordUnionDupe.purs

```diff
@@ -0,0 +1,12 @@
+module Main where
+
+import Prelude
+import Data.Record (merge)
+import Control.Monad.Eff.Console (log, CONSOLE)
+import Control.Monad.Eff (Eff)
+import Unsafe.Coerce (unsafeCoerce)
+
+main :: Eff (console::CONSOLE) Unit
+main = do
+  let result = (merge {left:1} {left:"Done"}).left
+  log $ unsafeCoerce result
```

Comment:
@doolse Row unions which happen by unification are always left-biased, so I think it makes sense to do the same thing with the `RowUnion` constraint. As you say, you would have to change the signature of `merge` to get the correct behavior otherwise.